content/facets.md

```diff
@@ -0,0 +1,138 @@
+# 12 Factor Facets
+
+While the 12 factors serve as helpful specific guidelines for development teams, the rationale behind the factors is to enable a set of application-platform interactions which simplify the operation of applications which follow the recommendations in the factors.  These interactions can be broadly grouped by the platform behaviors they enable, which serve as “facets” of operation where the 12 factors simplify maintenance.  The facets enable applications which are:
+
+* **Continuously Deployable**: it is easy to deploy new software versions in existing environments
+* **Composable**: it is easy to set up existing software versions in new environments
+* **Scalable**: it is easy to manage the software within an environment
+* **Observable**: it is easy to debug the software when running in an production environment
+
+The following sections will map how specific factors contribute to the different facets of application operations.  Applications which apply the contributing factors for a particular facet may be able to derive some benefits from the 12 factors even if they cannot satisfy all of the factors.
+
+# Continuously Deployable
+
+## What
+
+Updateable applications are able to automatically deploy software changes without manual intervention.  These practices have largely been codified as Continuous Integration / Continuous Delivery (CI/CD), but applications which adhere to the updateable facets are able to be *automatically* continuously delivered on an enabling platform without development teams needing to build application-specific deployment pipelines.
```

Comment:
The "application-specific" bit is doing a lot of lifting here. The key point is that you don't have to heavily customize the pipeline, but it could be read as a pipeline is not needed at all

Reply:
I'm not sure what you're suggesting here -- I was trying to cover things like "`git push` to deploy" on Heroku, where the platform sets up the git repo and the surrounding processes such that apps can simply push and deploy without needing to explicitly set up a pipeline.